frontend/src/components/yugioh/YugiohCardImage.tsx

```diff
@@ -0,0 +1,20 @@
+import { useTheme } from '@mui/material';
+
+type YugiohCardImageProps = {
+  src: string;
+};
+
+function YugiohCardImage(props: YugiohCardImageProps): JSX.Element {
+  const theme = useTheme();
+  const secondary = theme.palette.secondary.main;
+
+  return (
+    <div
+      className={`bg-white p-8 w-[314px] h-[422px] border-[${secondary}] border rounded flex justify-center items-center`}
```

Comment:
`rounded` -> `rounded-lg` to match the table

Reply:
This is done